BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -193,6 +222,24 @@ extension StoresTableViewController {
         
     }
     
+    override func tableView(_ tableView: UITableView, didHighlightRowAt indexPath: IndexPath) {
+        
+        guard tableView.cellForRow(at: indexPath) is StoresTableViewCell else { return }
```

Comment:
You can use do `guard let cell = ... as? StoresTableViewCell else { return }`, and that will take care of checking and casting in one shot.